client/src/components/ProductCard.tsx

```diff
@@ -143,6 +145,31 @@ function ProductCard ({ product, bookmark, log }: Props) {
     }
   };
 
+  const handleShareClick: React.MouseEventHandler<HTMLDivElement> = async () => {
+    const projectURL = 'https://oriori.fly.dev/home';
+    try{
+        if(navigator.share){
```

Comment:
Is this the code to invoke a device's native share flow? I wonder if it's possible to include the image url in the shared data 😀

Reply:
This we needed more confirmation. I looked online, some source stated modern devices has navigator.share() as its method to call the device's native share sheet function